libraries/Netdump/src/PacketType.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+    PacketType.cpp
+
+    Created on: 19 nov. 2019
+        Author: Herman
+*/
+
+#include <PacketType.h>
+
+namespace NetCapture
+{
+
+PacketType::PacketType()
+{
+}
+
+PacketType::~PacketType()
+{
+}
```

Comment:
No need to implement default constructor/destructor, the compiler provides them.